recipes/hmmlearn/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hmmlearn" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 694646f8302bc6402925a4b6892f3a5ccede06d25f22157c18cfbdecdb748361
+
+build:
+  number: 0
+  skip: True  # [py37]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - matplotlib >=1.1.1
+    - scikit-learn >=0.16,<0.20.0
+    - numpy >=1.9.3
```

Comment:
This since is building against numpy, the number version should be pinned in the host section and the run section should use a pin_compatible function, see the [Building Against NumPy](https://conda-forge.org/docs/meta.html#building-against-numpy) section in the docs for details.

Reply:
Can you please fix this before the next review?